libcloud/test/compute/test_ecs.py

```diff
@@ -943,6 +943,15 @@ def _delete_sg_by_id_DeleteSecurityGroup(self, method, url, body, headers):
         resp_body = self.fixtures.load('delete_security_group_by_id.xml')
         return (httplib.OK, resp_body, {}, httplib.responses[httplib.OK])
 
+    def _modify_sg_by_id_ModifySecurityGroup(self, method, url, body, headers):
+        params = {'RegionId': self.test.region,
+                  'SecurityGroupId': 'sg-fakeSecurityGroupId',
+                  'SecurityGroupName': 'sg-fakeSecurityName',
+                  'Description': 'sg-fakeSecurityDescription'}
+        self.assertUrlContainsQueryParams(url, params)
+        resp_body = self.fixtures.load('modify_security_group_by_id.xml')
+        return (httplib.OK, resp_body, {}, httplib.responses[httplib.OK])
+
```

Comment:
This is only a fixture, not a test: test methods start with `test_`. You need to ensure such `ex_modify_security_group_by_id` is called from the tests. codecov should have telled us this, I don't know why it's not part of the checks here.